### PR TITLE
OCM-7141 | ci: Automate 72451/72504/72505/72507/72509/72510

### PR DESCRIPTION
**Pull request by radtriste**
**What this PR does / why we need it**:
Implement day2 test cases

<details>
<summary>72504</summary>
$ ginkgo -focus 72504 tests/e2e/
time="2024-04-04T16:44:09+02:00" level=info msg="Running against env staging with gateway url https://api.stage.openshift.com"
time="2024-04-04T16:44:09+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
Running Suite: e2e tests suite - /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e
====================================================================================================
Random Seed: 1712241848

Will run 1 of 67 specs
time="2024-04-04T16:44:09+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
time="2024-04-04T16:44:09+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
time="2024-04-04T16:44:12+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
time="2024-04-04T16:44:14+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp"
time="2024-04-04T16:44:14+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/aws/vpc"
time="2024-04-04T16:44:17+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/aws/vpc"
time="2024-04-04T16:44:19+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp with args [-var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var name=np-72504-ge -var url=https://api.stage.openshift.com -var machine_type=m5.2xlarge -var replicas=3 -var autoscaling_enabled=false -var subnet_id=subnet-04d27a3aa93f76c91 -var auto_repair=true]"
time="2024-04-04T16:44:53+02:00" level=info msg="Running terraform destroy against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp"
args:  /usr/bin/terraform destroy -auto-approve -no-color -var replicas=3 -var autoscaling_enabled=false -var subnet_id=subnet-04d27a3aa93f76c91 -var auto_repair=true -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var name=np-72504-ge -var url=https://api.stage.openshift.com -var machine_type=m5.2xlarge
time="2024-04-04T16:44:56+02:00" level=info msg="Running terraform destroy against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp"
args:  /usr/bin/terraform destroy -auto-approve -no-color -var name=np-72504-ge -var url=https://api.stage.openshift.com -var machine_type=m5.2xlarge -var replicas=3 -var autoscaling_enabled=false -var subnet_id=subnet-04d27a3aa93f76c91 -var auto_repair=true -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em
•SSSSSSSSSSSSSSSSS
------------------------------
P [PENDING]
TF Test Verfication/Post day 1 tests Author:xueli-Critical-OCP-69145 @OCP-69145 @xueli Apply to change security group will be forbidden [NonHCPCluster]
/home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e/verfication_post_day1_test.go:262
------------------------------
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Ran 1 of 67 Specs in 47.970 seconds
SUCCESS! -- 1 Passed | 0 Failed | 1 Pending | 65 Skipped
PASS
</details>

<details>
<summary>72505</summary>
$  ginkgo -focus 72505 tests/e2e/
time="2024-04-05T09:02:26+02:00" level=info msg="Running against env staging with gateway url https://api.stage.openshift.com"
time="2024-04-05T09:02:26+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
Running Suite: e2e tests suite - /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e
====================================================================================================
Random Seed: 1712300544

Will run 1 of 67 specs
time="2024-04-05T09:02:26+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
time="2024-04-05T09:02:26+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
time="2024-04-05T09:02:29+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
SSSSSSSSSSStime="2024-04-05T09:02:31+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp"
time="2024-04-05T09:02:31+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/aws/vpc"
time="2024-04-05T09:02:34+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/aws/vpc"
time="2024-04-05T09:02:36+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp with args [-var url=https://api.stage.openshift.com -var autoscaling_enabled=true -var min_replicas=2 -var subnet_id=subnet-04d27a3aa93f76c91 -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var name=np-72505-b2 -var machine_type=m5.xlarge -var max_replicas=4 -var auto_repair=true]"
time="2024-04-05T09:02:41+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp with args [-var min_replicas=1 -var auto_repair=true -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var autoscaling_enabled=true -var machine_type=m5.xlarge -var max_replicas=3 -var subnet_id=subnet-04d27a3aa93f76c91 -var name=np-72505-b2 -var url=https://api.stage.openshift.com]"
time="2024-04-05T09:02:45+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp with args [-var autoscaling_enabled=false -var subnet_id=subnet-04d27a3aa93f76c91 -var auto_repair=true -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var name=np-72505-b2 -var url=https://api.stage.openshift.com -var machine_type=m5.xlarge -var replicas=3]"
time="2024-04-05T09:02:48+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp with args [-var name=np-72505-b2 -var url=https://api.stage.openshift.com -var machine_type=m5.xlarge -var replicas=4 -var autoscaling_enabled=false -var subnet_id=subnet-04d27a3aa93f76c91 -var auto_repair=true -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em]"
time="2024-04-05T09:02:52+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp with args [-var machine_type=m5.xlarge -var replicas=0 -var autoscaling_enabled=false -var subnet_id=subnet-04d27a3aa93f76c91 -var auto_repair=true -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var name=np-72505-b2 -var url=https://api.stage.openshift.com]"
time="2024-04-05T09:02:55+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp with args [-var auto_repair=true -var name=np-72505-b2 -var machine_type=m5.xlarge -var autoscaling_enabled=true -var max_replicas=2 -var subnet_id=subnet-04d27a3aa93f76c91 -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var url=https://api.stage.openshift.com -var min_replicas=1]"
time="2024-04-05T09:03:00+02:00" level=info msg="Running terraform destroy against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp"
args:  /usr/bin/terraform destroy -auto-approve -no-color -var name=np-72505-b2 -var min_replicas=1 -var subnet_id=subnet-04d27a3aa93f76c91 -var auto_repair=true -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var url=https://api.stage.openshift.com -var machine_type=m5.xlarge -var autoscaling_enabled=true -var max_replicas=2
•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
------------------------------
P [PENDING]
TF Test Verfication/Post day 1 tests Author:xueli-Critical-OCP-69145 @OCP-69145 @xueli Apply to change security group will be forbidden [NonHCPCluster]
/home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e/verfication_post_day1_test.go:262
------------------------------
SSSSSSSSSSSSSSSSSSSSS

Ran 1 of 67 Specs in 36.634 seconds
SUCCESS! -- 1 Passed | 0 Failed | 1 Pending | 65 Skipped
PASS

</details>

<details>
<summary>72451</summary>
$ ginkgo -focus 72451 tests/e2e/
time="2024-04-05T11:07:06+02:00" level=info msg="Running against env staging with gateway url https://api.stage.openshift.com"
time="2024-04-05T11:07:06+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
Running Suite: e2e tests suite - /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e
====================================================================================================
Random Seed: 1712308025

Will run 1 of 69 specs
time="2024-04-05T11:07:06+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
time="2024-04-05T11:07:06+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
time="2024-04-05T11:07:10+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
Stime="2024-04-05T11:07:11+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
time="2024-04-05T11:07:11+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
time="2024-04-05T11:07:15+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp with args [-var url=https://api.stage.openshift.com -var aws_region=us-west-2 -var custom_properties={\"custom_property\":\"test\",\"nothing\":\"\",\"qe_usage\":\"\",\"some\":\"thing\"}]"
time="2024-04-05T11:07:23+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp with args [-var url=https://api.stage.openshift.com -var aws_region=us-west-2 -var custom_properties={\"custom_property\":\"test\",\"nothing\":\"\",\"qe_usage\":\"\",\"some\":\"thing2\"}]"
time="2024-04-05T11:07:30+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp with args [-var url=https://api.stage.openshift.com -var aws_region=us-west-2 -var custom_properties={\"custom_property\":\"test\",\"qe_usage\":\"\"}]"
time="2024-04-05T11:07:37+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp with args [-var url=https://api.stage.openshift.com -var aws_region=us-west-2 -var custom_properties={\"custom_property\":\"test\",\"qe_usage\":\"\"}]"
•SSSSSSSSSSSSS
------------------------------
P [PENDING]
TF Test Verfication/Post day 1 tests Author:xueli-Critical-OCP-69145 @OCP-69145 @xueli Apply to change security group will be forbidden [NonHCPCluster]
/home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e/verfication_post_day1_test.go:262
------------------------------
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Ran 1 of 69 Specs in 35.712 seconds
SUCCESS! -- 1 Passed | 0 Failed | 1 Pending | 67 Skipped
PASS

</details>

<details>
<summary>72507</summary>
$ ginkgo -focus 72507 tests/e2e/
time="2024-04-05T10:56:20+02:00" level=info msg="Running against env staging with gateway url https://api.stage.openshift.com"
time="2024-04-05T10:56:20+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
Running Suite: e2e tests suite - /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e
====================================================================================================
Random Seed: 1712307379

Will run 1 of 69 specs
time="2024-04-05T10:56:20+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
time="2024-04-05T10:56:20+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
time="2024-04-05T10:56:23+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
SSSSSSSSSSSSSSSSSSSSSSSS
------------------------------
P [PENDING]
TF Test Verfication/Post day 1 tests Author:xueli-Critical-OCP-69145 @OCP-69145 @xueli Apply to change security group will be forbidden [NonHCPCluster]
/home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e/verfication_post_day1_test.go:262
------------------------------
SSSSSSSStime="2024-04-05T10:56:25+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp"
time="2024-04-05T10:56:25+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/aws/vpc"
time="2024-04-05T10:56:28+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/aws/vpc"
time="2024-04-05T10:56:29+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp with args [-var url=https://api.stage.openshift.com -var machine_type=m5.xlarge -var autoscaling_enabled=false -var auto_repair=false -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var name=np-72507-te -var replicas=3 -var labels={\"l1\":\"v1\",\"l2\":\"v2\"} -var taints=[{\"key\":\"t1\",\"schedule_type\":\"NoSchedule\",\"value\":\"v1\"}] -var subnet_id=subnet-04d27a3aa93f76c91]"
time="2024-04-05T10:56:34+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp with args [-var name=np-72507-te -var url=https://api.stage.openshift.com -var machine_type=m5.xlarge -var replicas=3 -var labels={\"l3\":\"v3\"} -var taints=[{\"key\":\"t1\",\"schedule_type\":\"NoSchedule\",\"value\":\"v1\"},{\"key\":\"t2\",\"schedule_type\":\"NoExecute\",\"value\":\"\"}] -var subnet_id=subnet-04d27a3aa93f76c91 -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var autoscaling_enabled=false -var auto_repair=true]"
time="2024-04-05T10:56:38+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp with args [-var machine_type=m5.xlarge -var replicas=3 -var autoscaling_enabled=false -var subnet_id=subnet-04d27a3aa93f76c91 -var auto_repair=true -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var name=np-72507-te -var url=https://api.stage.openshift.com]"
time="2024-04-05T10:56:41+02:00" level=info msg="Running terraform destroy against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp"
args:  /usr/bin/terraform destroy -auto-approve -no-color -var autoscaling_enabled=false -var subnet_id=subnet-04d27a3aa93f76c91 -var auto_repair=true -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var name=np-72507-te -var url=https://api.stage.openshift.com -var machine_type=m5.xlarge -var replicas=3
•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Ran 1 of 69 Specs in 23.776 seconds
SUCCESS! -- 1 Passed | 0 Failed | 1 Pending | 67 Skipped
PASS

</details>

<details>
<summary>72509</summary>
$ ginkgo -focus 72509 tests/e2e/
time="2024-04-05T09:41:17+02:00" level=info msg="Running against env staging with gateway url https://api.stage.openshift.com"
time="2024-04-05T09:41:17+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
Running Suite: e2e tests suite - /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e
====================================================================================================
Random Seed: 1712302875

Will run 1 of 69 specs
time="2024-04-05T09:41:17+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
time="2024-04-05T09:41:17+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
time="2024-04-05T09:41:20+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
SSSSSSSSSSSSSS
------------------------------
P [PENDING]
TF Test Verfication/Post day 1 tests Author:xueli-Critical-OCP-69145 @OCP-69145 @xueli Apply to change security group will be forbidden [NonHCPCluster]
/home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e/verfication_post_day1_test.go:262
------------------------------
SSSSSSSSSSSSSSSSSSStime="2024-04-05T09:41:22+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp"
time="2024-04-05T09:41:22+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/aws/vpc"
time="2024-04-05T09:41:25+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/aws/vpc"
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Ran 0 of 69 Specs in 9.513 seconds
SUCCESS! -- 0 Passed | 0 Failed | 1 Pending | 68 Skipped
PASS

</details>

<details>
<summary>72510</summary>
$ ginkgo -focus 72510 tests/e2e/
time="2024-04-05T09:24:17+02:00" level=info msg="Running against env staging with gateway url https://api.stage.openshift.com"
time="2024-04-05T09:24:17+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
Running Suite: e2e tests suite - /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e
====================================================================================================
Random Seed: 1712301856

Will run 1 of 69 specs
time="2024-04-05T09:24:17+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
time="2024-04-05T09:24:17+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
time="2024-04-05T09:24:21+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
------------------------------
P [PENDING]
TF Test Verfication/Post day 1 tests Author:xueli-Critical-OCP-69145 @OCP-69145 @xueli Apply to change security group will be forbidden [NonHCPCluster]
/home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e/verfication_post_day1_test.go:262
------------------------------
SSSSSSSSSSSSSSSSSSSSSSSSSSStime="2024-04-05T09:24:22+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp"
time="2024-04-05T09:24:22+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/aws/vpc"
time="2024-04-05T09:24:25+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/aws/vpc"
time="2024-04-05T09:24:27+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp with args [-var subnet_id=subnet-04d27a3aa93f76c91 -var auto_repair=true -var url=https://api.stage.openshift.com -var replicas=3 -var machine_type=m5.2xlarge -var autoscaling_enabled=false -var tags={\"aaa\":\"bbb\",\"ccc\":\"ddd\"} -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var name=np-72510-j5]"
time="2024-04-05T09:24:32+02:00" level=info msg="Running terraform destroy against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp"
args:  /usr/bin/terraform destroy -auto-approve -no-color -var auto_repair=true -var cluster=2ae5mq021okjdp0sb3e7l32r7ald84em -var url=https://api.stage.openshift.com -var replicas=3 -var subnet_id=subnet-04d27a3aa93f76c91 -var name=np-72510-j5 -var machine_type=m5.2xlarge -var autoscaling_enabled=false -var tags={"aaa":"bbb","ccc":"ddd"}
•SSSSSSS

Ran 1 of 69 Specs in 18.249 seconds
SUCCESS! -- 1 Passed | 0 Failed | 1 Pending | 67 Skipped
PASS

</details>

**Which issue(s) this PR fixes** 
https://issues.redhat.com/browse/OCM-7141

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Build
- [x] CI
- [ ] Documentation
- [ ] Performance
- [ ] Refactor
- [ ] Style
- [ ] Unit tests
- [ ] Subsystem tests

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.

### CODE REVIEW

File: tests/utils/helper/assert.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+// if e is  Nil, will print expected error info and then case fails.
+
+func AssertWaitPollWithErr(e error, msg string) {
+	if e != nil {
+		Logger.Infof("the error: %v", e)
+		return
+	}
+
+	err := fmt.Errorf("case: %v\nexpected error not got because of %v", CurrentSpecReport().FullText(), msg)
+	Expect(err).NotTo(HaveOccurred())
```
Comment (xueli181114) [inline tests/utils/helper/assert.go:48]: If we use Expect out of testing specs we have to `defer GinkgoRecover()` for all test suites. Are that expected?

Comment (radtriste) [reply tests/utils/helper/assert.go:48]: Don't we ? 
I just copied this piece of code from ROSA CLI, I guess I can tweak it a bit :)

Comment (radtriste) [reply tests/utils/helper/assert.go:48]: ok that is what I thought. I only missed that one file for the `defer GinkgoRecover()`. I added it

**Review by ciaranRoche — APPROVED** [review]
/lgtm

**Pull request merged**